src/io/flutter/utils/AndroidUtils.java

```diff
@@ -114,4 +185,451 @@ private static String validateName(String name, boolean requiresSeparator) {
     }
     return hasSep || !requiresSeparator ? null : "The package must have at least one '.' separator";
   }
+
+  /*
+    Add-to-app notes:
+    - The Flutter module must be added to the Android app via manual editing as in the add-to-app docs,
+      or by using the forthcoming tool in Android Studio that generates a module and does the editing.
```

Comment:
If I understand what this is potentially referring to, this is already possible via the new section in the wiki https://github.com/flutter/flutter/wiki/Add-Flutter-to-existing-apps#1-depend-on-the-android-archive-aar. Though tracing back to the source of the AAR is indeed going to be a challenge.

Reply:
Thanks for taking a look, @xster.